Ginger/GingerCoreNET/Database/NoSqlBase/GingerCouchbase.cs

```diff
@@ -153,25 +153,35 @@ public bool ConnectToBucket(string bucketName)
         private string GetBucketName(string inputSQL)
         {
             string bucketName = string.Empty;
-            string inputSQLLower = inputSQL.ToLower();
-            int bucket1 = 0;
-            int bucket2 = 0;
+            string[] bucketNameArray;
             if (Action == ActDBValidation.eDBValidationType.RecordCount)
             {
                 bucketName = inputSQL.Replace("`", "");
```

Comment:
Line no 167- we are making inputSQL.ToLower() and getting index of "Update" this will never pass since the word "update" would be lower and we should find indexOf("update")

Also for Update word we are comparing "Update " and for "from" keyword we are comparing " from ",  keep it same for both i.e. any keyword should be separated by space before and after. Lets expect users to follow the syntax.

Also we discussed to have proper error messages to users in case of incorrect syntax entered by user, instead of failing the SQL or failure without any user friendly message, please handle it.